crates/swc_common/src/syntax_pos/hygiene.rs

```diff
@@ -15,6 +15,7 @@
 //! and definition contexts*. J. Funct. Program. 22, 2 (March 2012), 181-216.
 //! DOI=10.1017/S0956796812000093 <https://doi.org/10.1017/S0956796812000093>
 
+#[allow(unused)]
```

Comment:
These are for the cases swc_common built with plugin-mode feature, where host fn's execution path is deactivated.